common-app-lib/src/cmr/common_app/api/health.clj

```diff
@@ -53,5 +55,7 @@
   takes a request-context as a parameter to determine if the application and its dependencies are
   working as expected."
   [health-fn]
+  (OPTIONS "/" req common-routes/options-response)
```

Comment:
No need to add this route.

Reply:
oh, I didn't know that. Then we need both.

Reply:
So, the `/` paths are in each individual app (search, ingest, etc.), not in common-app-lib.

These changes are being made in common-app-lib for expediency and to support re-use ... if we just stick to supporting `/health` right now, we can do this as planned.

If we also support `/`, then we'd have to update the other apps, too. We might want to hold off on that until we get an explicit request for that ...

Reply:
In that case, I'm going to remove OPTIONS "/" in this ticket.  I prefer to separate the support for real health checks (this ticket) and the support for "/" which is treated as health checks but is not real health checks.